test/gurka/gurka.cc

```diff
@@ -121,7 +121,14 @@ std::string build_valhalla_request(const std::vector<std::string>& location_type
 
   // we do this last so that options are additive/overwrite
   for (const auto& kv : options) {
-    rapidjson::Pointer(kv.first).Set(doc, kv.second);
+    // Check if the value is "true" or "false" and set as bool
+    if (kv.second == "true") {
+      rapidjson::Pointer(kv.first).Set(doc, true);
+    } else if (kv.second == "false") {
+      rapidjson::Pointer(kv.first).Set(doc, false);
```

Comment:
This is a bit of a hack, but currently there is no way to enable boolean flags in the `do_action` function options.

Reply:
It's a bit ugly but it should work using `"0"` or `"1"`

Reply:
You are right! I removed it.